GOVERNANCE.md

```diff
@@ -1,3 +1,175 @@
+confident reviewing a PR will typically call in additional core
+contributors to help with the review or provide additional thoughts.
+A core contributor who authors or merges a PR is typically responsible
+for any testing fallout caused by that PR, typically by involving the
+PR's author when it was not theirs.  In some cases, the reviewing core
+contributor or another volunteer may step up to help out.
+
+At present, the list of core contributors includes all of the
+technical leaders listed previously, as well as:
+
+* [Jade Abraham](https://github.com/jabraham17)
+* [Soohoon Choi](https://github.com/soohoonchoi)
+* [Matt Drozt](https://github.com/MattToast)
+* [Daniel Fedorin](https://github.com/DanilaFe)
+* [Michael Ferguson](https://github.com/mppf)
+* [Abhishek Girish](https://github.com/agirish)
+* [Shreyas Khandekar](https://github.com/ShreyasKhandekar)
+* [Vass Litvinov](https://github.com/vasslitvinov)
+* [David Longnecker](https://github.com/dlongnecke-cray)
+* [Ben McDonald](https://github.com/bmcdonald3)
+* [Ahmad Rezaii](https://github.com/arezaii)
+* [Anna Rift](https://github.com/riftEmber)
+* [Elliot Ronaghan](https://github.com/ronawho)
+* [Michelle Strout](https://github.com/mstrout)
+* [Karlon West](https://github.com/karlonw)
+
+
+Decision-Making
+---------------
+
+Generally speaking, decision-making on the Chapel project is done by
+the technical leadership and core contributors in this section on a
+consensus basis, taking the opinions of end-users into account when
+the decision is likely to affect them.
+
+For non-trivial internal decisions, or initial explorations into
+public decisions, the project typically forms an ad hoc subteam of at
+least three core contributors to explore the topic and propose a
+solution, where anyone on the team is encouraged to participate if
+they are interested.  Once the subteam has reached a conclusion, it is
+summarized back to the full team.  Generally speaking, those who did
+not join the subteam are expected to go along with the subteam's
+decision, unless they have an insight or strong reason that the
+proposal is untenable—particularly if it involves factors the subteam
+was not, or could not have been, aware of.  In practice, this has
+happened only very rarely.
+
+Decisions that relate to the design of the language or libraries are
+captured and discussed on public [GitHub
+issues](https://github.com/chapel-lang/chapel/issues?q=is%3Aissue) as
+a means of soliciting input from the community. Many of these issues
+are opened or requested by users directly. Decisions that are
+particularly impactful to user codes are advertised directly to users
+to get their feedback by pointing them to the GitHub issues or
+contacting them directly.  For some of the most impactful changes, we
+have run polls and worked hard to involve as many users as possible in
+order to avoid surprises and attempt to develop a consensus community
+view.
+
```

Comment:
I recommend that you say somewhere in this section that the technical leader has the ability to veto a decision and also serves as a final decision-maker in the event the community cannot come to a decision.